nautobot/extras/filters/__init__.py

```diff
@@ -493,6 +493,12 @@ class Meta:
 
 
 class ExternalIntegrationFilterSet(NautobotFilterSet):
+    q = SearchFilter(
```

Comment:
In the interim until #5068, should we add a basic `test_search` test case to `extras.tests.test_filters.ExternalIntegrationTestCase`?